api/http/server.go

```diff
@@ -1134,3 +1146,25 @@ func (lrw *loggingResponseWriter) WriteHeader(code int) {
 func isDecryptError(err error) bool {
 	return strings.Contains(err.Error(), api.ErrDecrypt.Error())
 }
+
+// periodicTagTrace queries the tag every 2 seconds and logs its state to the span
+func periodicTagTrace(tags *chunk.Tags, tagUID uint32, q chan struct{}, sp opentracing.Span) {
```

Comment:
I think this function is no longer needed, because we instrumented `push sync` to link the intermediate spans with the root `http post` span. If you search the codebase, you will see that nothing refers to it.